### DISCUSSION TITLE
To to use `prefers-color-scheme` instead of the current system?

### DISCUSSION THREAD

**Discussion by babakfp** [huntabyte/shadcn-svelte#426] [General]
Who came up with the idea to blind users even when there is a dark mode available?
The dark mode should kick in by default if the user has set it in their `prefers-color-scheme`.
I removed `darkMode: ["class"]` but it didn't change anything.

**Comment by huntabyte** [comment]
https://www.shadcn-svelte.com/docs/dark-mode
